docs/ci/ci-automated-release-workflow.md

```diff
@@ -0,0 +1,120 @@
+# Creating Automated Releases: Design Document
+
+## Motivation & Overview
+
+Presently, the release processes for every library within this `instructlab` GitHub organization is entirely manual:
+
+A maintainer has to:
+
+1. Manually create a new release branch -- e.g., `release-0.y.0`,
```

Comment:
Is this document/automation only meant to cover the y stream releases?

Reply:
Yes, y-stream only. Sorry, I should clarify that!

At this point, I don't think it's worth automating z-stream releases. We can certainly consider automating them in the future, but they can be complicated due to the fact sometimes backporting and manual rebasing is needed.

Reply:
I just updated the document to specify y-stream. Can you take a look?